docs/0.1/glossary.md

```diff
@@ -9,45 +9,141 @@
 This glossary defines Hyperledger Grid terms and concepts.
 <br><br>
 
+<h3 class="glossary-header" id="agent">
+agent
+</h3>
+<p class="glossary-definition">
+Person or entity that acts on behalf of a Pike organization to create and manage
+items such as Product or Location records. Pike roles (also called
+"permissions") control what an agent is allowed to do, such as creating schemas
+or product records.
+</p>
 
-<h3 class="glossary-header" id="distributed_ledger">
-Distributed ledger
+<h3 class="glossary-header" id="batch">
+batch
+</h3>
+<p class="glossary-definition">
+Collection of related transactions that are handled as a group by the backend
+distributed ledger.
+(For more information, see the <a
+href="https://sawtooth.hyperledger.org/docs/core/releases/latest/glossary.html">
+Sawtooth glossary</a>.)
+The Grid REST API provides the <code>batches</code> endpoint to submit batches
+of transactions and the <code>batch_statuses</code> endpoint to query the commit
+status of submitted batches. In addition, the <code>grid</code> CLI includes
+subcommands that can submit batches of transactions from the command line.
+</p>
+
+<h3 class="glossary-header" id="circuit">
+circuit
 </h3>
+<p class="glossary-definition">
+(Grid on Splinter) Virtual network that safely and securely enforces privacy
+scope boundaries. A circuit defines the scope and visibility domains for two
+or more connected nodes.
+</p>
 
+<h3 class="glossary-header" id="distributed_ledger">
+distributed ledger
+</h3>
 <p class="glossary-definition">
 Distributed database that records transactions, in chronological order,
 shared by all participants in a network. (Also called a "blockchain".)
 Each block is linked by a cryptographic hash to the previous block.
 </p>
 
+<h3 class="glossary-header" id="grid_daemon_gridd">
+Grid daemon (gridd)
+</h3>
+<p class="glossary-definition">
+Daemon process that provides services such as the Grid REST API and state delta
+export functionality, which integrates with the backend validator to materialize
```

Comment:
are we calling splinter "validator" or should this be changed to be backend generic?

Reply:
Changed to "backend distributed ledger".